examples/ucnf-kiknos/helm/kiknos_vpn_endpoint/templates/kiknos-nse-ucnf.yaml

```diff
@@ -188,12 +192,18 @@ metadata:
   name: ucnf-kiknos-{{ .Values.nsm.serviceName }}
 data:
   config.yaml: |
+    {{- if .Values.ucnf.natIP }}
+    initactions:
+    - dpconfig:
+        nat44pools:
```

Comment:
Does this key map to: https://github.com/ligato/vpp-agent/blob/master/proto/ligato/vpp/nat/nat.proto#L126-L136

I guess the new build of the vppagent image creates the external nat44Interface config.

Reply:
yes, it refers to the `Nat44AddressPool` proto that you referenced